test/integration/select_test.lua

```diff
@@ -805,3 +806,55 @@ add('test_select_with_collations', function(g)
     local objects = crud.unflatten_rows(result.rows, result.metadata)
     t.assert_equals(objects, get_by_ids(customers, {2, 4}))
 end)
+
+if crud_utils.tarantool_supports_uuids() then
```

Comment:
Look at `skip_if` function of `luatest`.

Reply:
Added skip_if